product-manual/audiences.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Audiences"
-description: "Learn more about Audiences."
+description: "Segment your visitors and accounts by almost anything."
```

Comment:
Happy to use a different word if you don't like "segment":

Some alternatives:
- cohort
- subset
- group
- ??

Reply:
I like segment fwiw!